external-import/tenable-security-center/tenable_security_center/adapters/tsc_api/v5_13_common.py

```diff
@@ -0,0 +1,353 @@
+"""Provide Common classes and functions for the Tenable Security Center Interface using the Rest API >= 5.13."""
+
+# isort: skip_file
+# isort is removing the type ignore untyped import comment conflicting with mypy
+
+import datetime
+import re
+
+from typing import Any, Callable, Optional
+
+from pydantic import BaseModel, ConfigDict, Field, AwareDatetime
+
+
+ASSETS_CHUNK_SIZE = 10
+FINDINGS_CHUNK_SIZE = 100
+
+FINDING_FILTERS = (
```

Comment:
after second thought, it might be better to move these filters into README

Reply:
Thanks for you remark, As it is likely to evolve I removed it